developers/dev-faq.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Development FAQ
+description: Commonly Asked Questions Regarding the Technical Aspects of the Ocean Stack
```

Comment:
Change "Commonly Asked Questions Regarding the Technical Aspects of the Ocean Stack" 
to "Frequently Asked Questions About Ocean Technology"

Why: FAQ acronym should unpack into the proper words. And overall it needed shortening.